docs/development/getting_started.rst

```diff
@@ -51,7 +51,7 @@ If you're using Ubuntu, you can install a sufficiently recent version of
 Ansible from backports (if you prefer): ``sudo apt-get install
 ansible/trusty-backports``
 
-.. note:: Tested: Ansible 1.8.4
+.. note:: Tested: Ansible 1.9.4
```

Comment:
I've spun up the staging environment successfully using Ansible 1.9.4. That's what I was running when I found the expired key, and later got through the whole playbook. Is that sufficient?